packages/amplify-e2e-tests/src/__tests__/rds-refers-to.test.ts

```diff
@@ -45,9 +39,9 @@ describe('RDS Relational Directives', () => {
 
   beforeAll(async () => {
     projRoot = await createNewProjectDir('rdsreferstoapi1');
-    await setupDatabase();
     await initProjectAndImportSchema();
     await amplifyPush(projRoot);
+    await sleep(2 * 60 * 1000); // Wait for 2 minutes for the VPC endpoints to be live.
```

Comment:
Does [`DescribeVpcEndpoints`](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeVpcEndpoints.html) correctly report status? I'd love not to spend 2 minutes in every test waiting for an endpoint to get created

Reply:
Based on last few E2E test runs, it is observed that it takes about 20-30 seconds for the VPC endpoints to be active even though the status shows 'AVAILABLE'. This delay was added to avoid flaky tests.